src/js/languages/default.json

```diff
@@ -93,6 +93,22 @@
     "multiple": "multiple",
     "selected": "; Selected: {currentSelectedValue}"
   },
+  "selectMultiple": {
+    "summarizedValue": "{selected} of {total} selected",
+    "clearAll": "Clear All",
+    "selectAll": "Select All",
+    "selected": "{selected} selected",
+    "selectedOfTotal": "{selected} of {total} selected",
+    "onMore": "+ {showMoreCount} more options",
+    "clearAllA11y": "{selected} options selected. Clear all?",
+    "selectAllA11y": "Select all {total} options",
+    "selectDropDown": "Select dropdown",
+    "open": "Open Drop",
```

Comment:
```suggestion
    "selectAll": "Select All",
```

Reply:
I'm not following this suggestion @britt6612 . Is this still relevant?

Reply:
no longer relevant